packages/realm-network-transport/src/types.ts

```diff
@@ -52,3 +52,122 @@ export interface NetworkTransport {
         handler: ResponseHandler,
     ): void;
 }
+
+// AbortController
+
+type AbortSignal = any;
+
+/** A controller object that allows you to abort one or more DOM requests as and when desired. */
+
+export type AbortController = {
```

Comment:
I agree that this is a little stupid. The `abort-controller` package does not ship with types and there are no types available via a "definitely typed" @types package, so ...
I put the types/abort-controller/index.d.ts in, but it's not really a strong guarantee that this will match the package's implementation.

Reply:
Is there any advantage over just using `any`? It seems like either way we are just telling the compiler "trust me, I know what I'm doing", this is just less explicit about doing that.